expr/consolidations/consolidations.go

```diff
@@ -35,6 +40,18 @@ var ConsolidationToFunc = map[string]func([]float64) float64{
 
 var AvailableSummarizers = []string{"sum", "total", "avg", "average", "avg_zero", "max", "min", "last", "current", "range", "rangeOf", "median", "multiply", "diff", "count", "stddev"}
```

Comment:
I think `first` should be also added to the list here as well as you've implemented it just few lines below.

Reply:
Done